docs/guide/quickstart.qmd

```diff
@@ -0,0 +1,763 @@
+It is therefore expected that the confluence basin reaches a level where the outflow is equal to the inflow via the rating curve.
+Only then is the confluence basin in equilibrium.
+To ensure that inflow equals outflow (IN=OUT) and keeping in mind the maximum depth of the river is 6 m, the Q-h relationship in @tbl-input2 will be used as input.
+
++---------------------+--------------------+
+| Water Level (h) [m] | Outflow (Q) [m³/s] |
++=====================+====================+
+|         0.0         |         0.0        |
++---------------------+--------------------+
+|          2          |         50         |
++---------------------+--------------------+
+|          5          |         200        |
++---------------------+--------------------+
+
+: Input data for the Tabulated Rating Curve {#tbl-input2}
+
+In Ribasim, the Q-h relation is a linear function, so the points in between will be linearly interpolated.
+@fig-discharge illustrates the visual process and shows a progressive increase in discharge with rising water levels.
+In this case this means:
+
+- At level 0: No discharge occurs. This represents a condition where the water level is too low for any flow to be discharged.
+- At level 2 : Discharge is max. 50m3/s. This is a bit above the average discharge rate, corresponding to the water level where normal flow conditions are established.
+- At level 5: Discharge rate reaches 200m3/s. This  discharge rate occurs at the  water level during wet periods, indicating higher flow capacity.
+
+![Discharge at corresponding water levels](https://github.com/user-attachments/assets/14799746-3b4d-4d4c-9b8a-37fbe1d88081){fig-align="left" #fig-discharge}
+
+Taking this into account, add the Tabulated Rating Curve as follows:
+
+```python
+model.tabulated_rating_curve.add(
+    Node(4, Point(-1.5, -1.5), name='MainConf'),
+    [tabulated_rating_curve.Static(
+        level=[0.0, 2, 5],
+        flow_rate=[0.0, 50, 200],
+        )
+    ]
+)
+```
+### Step 6: Terminal node
+Finally all the water will discharge into the ocean.
+Schematize this with the terminal node as it portrays the end point of the model.
+Besides the node number/name and location, no further input is needed.
+
+```python
+model.terminal.add(Node(5, Point(-1.5, -3.0), name="Terminal"))
+```
+### Step 7: Defining edges
+Implement the connections (edges) between the nodes, in the following order:
+
+1.	Flow boundaries to the basin;
+2.	Basin to the rating curve;
+3.	Tabulated rating curve to the terminal.
+
+```python
+model.edge.add(model.flow_boundary[1], model.basin[3])
+model.edge.add(model.flow_boundary[2], model.basin[3])
+model.edge.add(model.basin[3], model.tabulated_rating_curve[4])
+model.edge.add(model.tabulated_rating_curve[4], model.terminal[5])
+```
+### Step 8: Visualization and model execution
+Plot the schematization, write the model configuration to the TOML file.
+Name the output file 'Crystal_1.1':
```

Comment:
```suggestion
Plot the schematization, write the model configuration to the `TOML` file.
Name the output file `Crystal_1.1`:
```

`Crystal_1.1` is not a file, did you mean to refer to `Crystal_1.1/ribasim.toml`?